src/styles/utils/_mixin.scss

```diff
@@ -0,0 +1,24 @@
+@mixin hover($_property, $_toValue) {
+  transition: #{$_property} 0.3s;
```

Comment:
Consider adding easing to the transition for a smoother effect, like 'ease-in-out'.